models/DataObject/Concrete.php

```diff
@@ -536,10 +534,12 @@ public function getValueFromParent($key, $params = null)
             $method = 'get' . $key;
             if (method_exists($parent, $method)) {
                 return call_user_func([$parent, $method], $params);
+            } else {
+                throw new \Exception(sprintf('Parent object does not have a method called `%s()`, unable to retrieve value for key `%s`', $method, $key));
             }
         }
 
-        return;
+        throw new \Exception('No parent object available to get a value from');
```

Comment:
It should throw a specific Exception to Pimcore to be able to specifically catch it

Reply:
Yep, makes sense, thanks!